src/App.js

```diff
@@ -35,6 +36,7 @@ class App extends Component {
                 <Route exact path="/register" component={RegistrationPage} />
                 <Route exact path="/login" component={LoginPage} /> 
                 <Route exact path="/dashboard" component={Dashboard} />
+                <Route exact path="/create-event" component={CreateEvent} />
```

Comment:
Lets use the newEventMain component here, then inside of newEventMain, insert component CreateEvent